src/main/resources/application-RECEIVER.properties

```diff
@@ -183,10 +188,10 @@ application.user.lock.duration=30
 # time unit used for locking user, possible values are: SECONDS,MINUTES,HOURS,DAYS
 application.user.lock.unit=MINUTES
 
-application.selfdescription.description=Connector description
-application.selfdescription.title=Connector title
-application.selfdescription.curator=http://curatorURI.com
-application.selfdescription.maintainer=http://maintainerURI.com
+application.selfdescription.description=Receiver Connector description
+application.selfdescription.title=Receiver Connector title
```

Comment:
Good one to differentiate also test property values

Reply:
thanks :+1: